src/Sql/Ddl/AlterTable.php

```diff
@@ -229,7 +230,10 @@ protected function processDropConstraints(PlatformInterface $adapterPlatform = n
     {
         $sqls = [];
         foreach ($this->dropConstraints as $constraint) {
-            $sqls[] = $adapterPlatform->quoteIdentifier($constraint);
+            $sqls[] = [
+                'CONSTRAINT',
+                $adapterPlatform->quoteIdentifier($constraint->getName())
```

Comment:
Until this https://github.com/zendframework/zend-db/pull/233 goes through that fixes bugs in quoteIdentifier vs quoteIdentifierChain this will break PostgreSQL. If lucky and that PR gets merged prior to this, then its fine.